### PR TITLE
[B5] hqcase

### PR DESCRIPTION
**Pull request by millerdev**
Migrate 'hqcase' app directly to Bootstrap 5 with `--no-split`.

Before:
explode_cases-b3

After:
explode_cases-b5

:fish: Review by commit.

### Safety story

UI-only changes, no JS changes.

### Automated test coverage

No.

### QA Plan

No QA.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: corehq/apps/hqcase/templates/hqcase/explode_cases.html [hunk 1]
```diff
@@ -1,76 +1,77 @@
...
+      <div class="col-md-8">
+        <div class="card border-warning mb-3">
+          <div class="card-header text-bg-warning">Warning!</div>
+          <div class="card-body">
+            The case exploder will multiply all cases in the user's restore  by the explode factor entered below. So, for example, if you're exploding a test user with 20 cases, and set the factor to be 200, it will actually give you <strong>4,000</strong> cases, NOT 200. Be careful! For this reason, it's best to create a new test user with only one case to explode.
           </div>
         </div>
-        <div class="panel panel-default">
-          <div class="panel-body">
+        <div class="card card-default mb-3">
```
Comment (orangejenny) [inline corehq/apps/hqcase/templates/hqcase/explode_cases.html:25]: Nitpick: `card-default` isn't a class. The tool ought to be updated.

Comment (orangejenny) [reply corehq/apps/hqcase/templates/hqcase/explode_cases.html:25]: https://github.com/dimagi/commcare-hq/pull/34669

File: corehq/apps/hqcase/templates/hqcase/explode_cases.html [hunk 2]
```diff
@@ -1,76 +1,77 @@
...
+              <button type="submit" class="btn btn-outline-danger"
                       data-bind="attr: {disabled: !user_id() || factor() < 2 || factor() > 150 }">Explode Cases</button>
             </form>
           </div>
         </div>
       </div>
     </div>
-  </div>
-  {% if request|toggle_enabled:"SYNC_SEARCH_CASE_CLAIM" %}
+  {% if True or request|toggle_enabled:"SYNC_SEARCH_CASE_CLAIM" %}
```
Comment (orangejenny) [inline corehq/apps/hqcase/templates/hqcase/explode_cases.html:56]:
```suggestion
  {% if request|toggle_enabled:"SYNC_SEARCH_CASE_CLAIM" %}
```

**Review by orangejenny — COMMENTED** [review]
There's a stray piece of debug code, everything else looks good.

File: corehq/apps/hqcase/templates/hqcase/explode_cases.html (hunk 2 above)
Comment (millerdev) [reply corehq/apps/hqcase/templates/hqcase/explode_cases.html:56]: :facepalm: Thank you.

File: corehq/apps/hqcase/templates/hqcase/explode_cases.html (hunk 1 above)
Comment (millerdev) [reply corehq/apps/hqcase/templates/hqcase/explode_cases.html]: https://github.com/dimagi/commcare-hq/pull/34668/commits/0d931a373129e5a0958c3e5db6fdada45a3e1983

**Review by orangejenny — APPROVED** [review]

**Pull request merged**